Bug: NPE at main.go+39

```
/tmp/___go_build_github_com_kyberorg_honeypot_cmd_honeypot --access-log=a.log --prom-metrics-enable --with-raw-metrics --raw-metrics-file=metrics.log
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x69ab26]

goroutine 1 [running]:
github.com/sirupsen/logrus.(*Logger).newEntry(0x0, 0x40ce7b)
	/home/asm/IdeaProjects/honeypot/vendor/github.com/sirupsen/logrus/logger.go:96 +0x26
github.com/sirupsen/logrus.(*Logger).Println(0x0, 0xc0000bff58, 0x2, 0x2)
	/home/asm/IdeaProjects/honeypot/vendor/github.com/sirupsen/logrus/logger.go:310 +0x2f
main.main()
	/home/asm/IdeaProjects/honeypot/cmd/honeypot/main.go:39 +0x21d
```

--- a/cmd/honeypot/config/broadcast.go
+++ b/cmd/honeypot/config/broadcast.go
@@ -7,10 +7,8 @@ var (
 )
 
 func init() {
-	once.Do(func() {
-		//init broadcaster
-		initLoginAttemptBroadcaster()
-	})
+	//init broadcaster
+	initLoginAttemptBroadcaster()
 }
 
 func GetLoginAttemptBroadcaster() *util.LoginAttemptBroadcaster {

--- a/cmd/honeypot/config/config.go
+++ b/cmd/honeypot/config/config.go
@@ -2,7 +2,6 @@ package config
 
 import (
 	"gopkg.in/alecthomas/kingpin.v2"
-	"sync"
 )
 
 //core flags
@@ -36,7 +35,6 @@ var (
 
 //internal vars
 var (
-	once      sync.Once
 	appConfig *applicationConfiguration
 )
 
@@ -74,10 +72,8 @@ type GeoIP struct {
 }
 
 func init() {
-	once.Do(func() {
-		//parse flags
-		kingpin.Parse()
-	})
+	//parse flags
+	kingpin.Parse()
 
 	appConfig = &applicationConfiguration{
 		Port:            *port,

--- a/cmd/honeypot/config/logger.go
+++ b/cmd/honeypot/config/logger.go
@@ -14,10 +14,8 @@ var (
 )
 
 func init() {
-	once.Do(func() {
-		initApplicationLogger()
-		initAccessLogger()
-	})
+	initApplicationLogger()
+	initAccessLogger()
 }
 
 //GetAccessLogger logger for access log

